gustaf/helpers/options.py

```diff
@@ -331,6 +331,23 @@ def __getitem__(self, key):
         else:
             raise TypeError(f"Invalid key type for {type(self)}")
 
+    def __call__(self, **kwargs):
+        """Alias to `update()`"""
+        self.update(**kwargs)
```

Comment:
Why do you want to add this alias? Would it not be different from how dictionaries normally work and might be misunderstood?

And does it improve the workflow to such a degree that possible/likely misunderstandings resulting from this diversion from the norm are worth it?

Reply:
good point. Maybe it is better to update `gustaf`/`splinepy` examples with `update()` instead of this. I will remove `__call__`